plugins/org/src/components/Cards/Group/GroupProfile/GroupProfileCard.tsx

```diff
@@ -191,7 +195,7 @@ export const GroupProfileCard = (props: { variant?: InfoCardVariants }) => {
                 secondary="Child Groups"
               />
             </ListItem>
-            <LinksGroup links={links} />
+            {props?.showLinks && <LinksGroup links={links} />}
```

Comment:
looking at the types `props` should be defined here, right? it's just `showLinks` that might be `undefined`.

```suggestion
            {props.showLinks && <LinksGroup links={links} />}
```